### ISSUE TITLE
1Password Importer not importing OTP Entries

### ISSUE THREAD

**Issue by drrk** [buttercup/buttercup-importer#71]
I'm trying to import from a 1Password 1pif file, but the OTP entries are not being brought across.  I have checked and they are in the 1pif file.

I'm quite following how the importer works, so not able to provide any possible solutions, but will continue to look at update if I can.

Sorry for the lack of detail in the bug report.

**Comment by perry-mitchell** [comment]
I don't recall, but it may be so that OTP items were not implemented in the import process yet. This could be added.

**Comment by drrk** [comment]
It does look like it - am I right that https://github.com/buttercup/buttercup-importer/blob/2782c39bbe581889b26c9e7554ab868caf8daac2/source/importers/OnePasswordImporter.js#L17 needs updating to map the OTP entry in the 1Password data to the Buttercup Vault otp entry?  I can investigate if I work that out and do some testing.

**Comment by perry-mitchell** [comment]
Yeah I think you're on the money there. One would need to detect an OTP property on the 1PW entry and create an associated property on the Buttercup side setting the property **value type** to be OTP. There's a type enum for this in the core.
